python/lsst/daf/butler/instrument.py

```diff
@@ -64,74 +62,72 @@ def _addCamera(self, registry):
         registry.addDataUnitEntry("Camera", {"camera": self.camera})
 
     def _addPhysicalFilters(self, registry):
+        print("REGISTERING PHYSICAL FILTER")
```

Comment:
I'm guessing you didn't intend to leave these in (more below).

Reply:
There are plenty of prints scattered about. I'll remove them before merging.

Reply:
They are mostly there so I could work out what logic was being triggered when.